src/ServiceControl.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -87,7 +87,7 @@ protected AcceptanceTest()
         [SetUp]
         public void Setup()
         {
-            port = FindAvailablePort(33333);
+            port = FindAvailablePort(33335);
```

Comment:
I'm trying everything at this point. This was to eliminate caching. This PR will never be merged.

So far it seems like the Nancy routing is behaving differently locally vs on TC. I have no idea why. The routes that match are different.